packages/igniter-runtime/lib/-private/promise.ts

```diff
@@ -0,0 +1 @@
+export default window.Promise;
```

Comment:
This seems weird though- RSVP promises will still schedule into the actions queue I expect, so native promises must be used for creating native microtasks. RSVP promises would create ember action queue items.